mmcls/core/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .evaluation import *  # noqa: F401, F403
+from .export import *  # noqa: F401, F403
```

Comment:
This importation should be removed. Otherwise, `onnxruntime` package would be a required package to be installed for the repo, which is unnecessary and unreasonable.